community/contributing/documentation_guidelines.rst

```diff
@@ -35,6 +35,19 @@ reference documentation about the reStructuredText markup language.
              main repository. These files are then later used to generate the
              in-editor documentation as well as the API reference of the
              online docs. Read more here: :ref:`doc_updating_the_class_reference`.
+             
+The 'Edit on Github' link
+-------------------------
+
+If you're reading documentation on ``docs.godotengine.org`` you'll see an 'Edit on GitHub' hyperlink at the top right of the page. Once you've created a GitHub account you can propose changes to a page you're reading as follows:
+
+1. Copy the URL that the GitHub link points to, and swap the version number (e.g. 3.1) for the word 'master'. e.g. ``https://github.com/godotengine/godot-docs/blob/master/community/contributing/docs_writing_guidelines.rst``
```

Comment:
Might want to mention that the version could be "latest", which might confuse the beginner if they're looking for a number.

Reply:
That makes sense. I added commits to this patch that I think fix that ambiguity.